contour/how-to-guides/index.hbs.md

```diff
@@ -0,0 +1,10 @@
+# Contour how-to guides
```

Comment:
nitpick: since this is a title, can we start every word with a capital letter (i.e. Contour How-to Guides)?